### PR TITLE
Adapt to changes in GNOME 3.32

### PR DESCRIPTION
**Pull request by ernestask**
See commits.

### CODE REVIEW

**Comment by ernestask** [comment]
Mmm… I should probably add a screenshot or two.

**Comment by ernestask** [comment]
Before:
Screenshot from 2019-05-24 16-41-02

After:
Screenshot from 2019-05-24 16-41-09

File: data/ui/oops-menus.ui [hunk 1]
```diff
@@ -38,4 +38,14 @@
       </item>
     </section>
   </menu>
+  <menu id="app_menu">
+    <item>
+      <attribute name="label" translatable="yes">_Preferences</attribute>
+      <attribute name="action">app.preferences</attribute>
+    </item>
+    <item>
+      <attribute name="label" translatable="yes">_About Problem Reporting</attribute>
```
Comment (mkutlak) [inline data/ui/oops-menus.ui]: Just keep the old `About`.

Comment (ernestask) [reply data/ui/oops-menus.ui]: Take a look at https://gitlab.gnome.org/GNOME/Initiatives/issues/4, it’s supposed to be changed.

Comment (mkutlak) [reply data/ui/oops-menus.ui]: <details>
<summary>About Problem Reporting</summary>

about_about

</details>

OK `¯\_(ツ)_/¯`

Comment (ernestask) [reply data/ui/oops-menus.ui]: You could argue for a better name, but there was a shift towards more generic ones: “Files”, “Photos”, etc.

**Comment by ernestask** [comment]
Any objections to this otherwise?

**Comment by ernestask** [comment]
So @mkutlak wanted to have more separation between the “Delete”/“Report” buttons and the application menu button. This got me thinking about cleaning up the header bar and achieving something akin to the picture in https://developer.gnome.org/hig/stable/sidebar-lists.html.en, i.e. having the left split of the header bar have just the title and menu button. The right side then would only have controls pertaining to the individual problem items.

**Comment by ernestask** [comment]
Dealing with the search bar is easy:

Screenshot from 2019-05-30 16-40-12

IMO, we shouldn’t hide it at all.

**Comment by ernestask** [comment]
The selection mode button could go away altogether. I was thinking of something like what is done in Contacts:

Screenshot from 2019-05-30 16-42-57

Screenshot from 2019-05-30 16-43-03

Screenshot from 2019-05-30 16-43-07

Which is very progressive-disclosure-like. On hover, the toggle button appears on the list box row, pressing which will activate the selection mode. Of course, we can’t do it exactly the same, because we have two columns of information, so it would probably have to be a permanent fixture on the left side.

**Comment by ernestask** [comment]
In any case, @mkutlak, for now, I just added a small margin to the menu button:

Screenshot from 2019-05-31 15-13-54

Thoughts?

**Comment by mkutlak** [comment]
LGTM.

**Pull request merged**